### PR TITLE
PRO-750+1043: Wallet Integration Feedback Changes

### PR DESCRIPTION
**Pull request by Yash-Chaubal**
Changes per feedback for https://linear.app/pillarproject/issue/PRO-750/buidler-wallet-functionality

### CODE REVIEW

File: src/components/TransactionBlock/Wallet/WalletNftsList.tsx [hunk 1]
```diff
@@ -210,6 +210,8 @@ const NftText = styled(Text)<{ marginTop?: number; size?: number }>`
   color: ${({ theme }) => theme.color.text.button};
   ${({ marginTop }) => `margin-top: ${marginTop || 4}px;`}
   ${({ size }) => `font-size: ${size || 12}px;`}
+  ${({ regular }) => !!regular && `font-family: "PTRootUIWebRegular", sans-serif;`}
+  ${({ medium }) => !!medium && `font-family: "PTRootUIWebMedium", sans-serif;`}
```
Comment (MayukhSamadder2000) [inline src/components/TransactionBlock/Wallet/WalletNftsList.tsx:214]: Shall we add the `regular` & `medium` booleans to the attributes of `NftText`?
```
const NftText = styled(Text)<{ marginTop?: number; size?: number, regular?: boolean, medium?: boolean }>
```

File: src/containers/Etherspot.tsx [hunk 2]
```diff
@@ -69,7 +64,7 @@ const Etherspot = ({
   smartWalletOnly,
   onLogout,
 }: EtherspotProps) => (
-  <ThemeProvider theme={merge({}, defaultTheme, themeOverride)}>
+  <ThemeProvider theme={merge({}, darkTheme, themeOverride)}>
```
Comment (MayukhSamadder2000) [inline src/containers/Etherspot.tsx:67]: Do we require the dark theme by default?

File: src/containers/Etherspot.tsx [hunk 3]
```diff
@@ -78,9 +73,9 @@ const Etherspot = ({
       smartWalletOnly={smartWalletOnly}
     >
       <style>
-        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Regular.css');
-        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Medium.css');
-        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Bold.css');
+        @import url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Regular.css'); @import
+        url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Medium.css'); @import
+        url('https://public.etherspot.io/buidler/fonts/PT-Root-UI_Bold.css');
```
Comment (MayukhSamadder2000) [inline src/containers/Etherspot.tsx:78]: Previous one looks better readability wise right? It's subjective so you may ignore this

**Review by MayukhSamadder2000 — APPROVED** [review]
LGTM! Just some comments wrt some minor issues if you can check them

File: src/components/TransactionBlock/Wallet/WalletNftsList.tsx (hunk 1 above)
Comment (Yash-Chaubal) [reply src/components/TransactionBlock/Wallet/WalletNftsList.tsx:214]: ah yes good catch

File: src/containers/Etherspot.tsx (hunk 2 above)
Comment (Yash-Chaubal) [reply src/containers/Etherspot.tsx:67]: yeah aldin asked for it on my ticket

File: src/containers/Etherspot.tsx (hunk 3 above)
Comment (Yash-Chaubal) [reply src/containers/Etherspot.tsx:78]: you know I completely agree, idk why the linter felt the need to change this

File: src/components/TransactionBlock/Wallet/WalletAssetsList.tsx [hunk 4]
```diff
@@ -90,7 +90,7 @@ const WalletAssetsList = ({
                             )}`}</ListItemText>
                           </ListItemLine>
 
-                          <ListItemLine>
+                          <ListItemLine margin_top={5}>
```
Comment (poocart) [inline src/components/TransactionBlock/Wallet/WalletAssetsList.tsx:93]: hmmm can we name this as camelCase to follow rest of codebase, thanks!

File: src/utils/theme.ts [hunk 5]
```diff
@@ -110,6 +114,9 @@ export const defaultTheme: Theme = {
       secondary: '#9889e4',
       selectInputRadioOn: '#ff7733',
       selectInputRadioOff: '#F8EFEA',
+      walletButton: 'linear-gradient(to bottom, #fd9250, #ff5548)',
+      walletChainDropdown: `#fff`,
```
Comment (poocart) [inline src/utils/theme.ts:118]:
```suggestion
      walletChainDropdown: '#fff',
```

**Review by poocart — APPROVED** [review]

**Pull request merged**